app/src/androidTest/java/com/example/myapplication/BikeRentalApiInstrumentedTest.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.myapplication
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import edu.usf.cutr.otp.bike_rental.api.BikeRentalApi
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers.Main
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import kotlin.coroutines.CoroutineContext
+
+@RunWith(AndroidJUnit4::class)
+class BikeRentalApiInstrumentedTest: CoroutineScope {
+    private lateinit var job: Job
+
+    override val coroutineContext: CoroutineContext
+        get() = job + Main
+
+    @Test
+    fun callBikeRentalApi() {
+        job = Job()
+
+        val bikeRentalApi = BikeRentalApi("https://raw.githubusercontent.com/CUTR-at-USF/opentripplanner-client-library/tests/app/src/androidTest/test_resources/bike_rental.json")
```

Comment:
Note that the `tests` part of these URLs will need to change to `master` prior to merging, as they won't be available in this branch after merge.